thoth/storages/graph/postgres.py

```diff
@@ -3475,6 +3475,17 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
+    def get_active_kebechet_github_installations_repos(self) -> Dict[str, List[str]]:
+        """Get all active repositories names with active Kebechet installation."""
+        with self._session_scope() as session:
+            active_installations = (
+                session.query(KebechetGithubAppInstallations)
+                .filter(KebechetGithubAppInstallations.is_active.is_(True))
+                .all()
+            )
+
+            return { i.slug:self.get_kebechet_github_installations_active_managers(slug=i.slug) for i in active_installations }
```

Comment:
?? we don't use one query inside another one the session needs to be one.

Reply:
So I've changed the function to return only repository names as originally intended, which should be proper to naming conventions

I hope I got it right this time @pacospace   ?

Reply:
Perfect, thanks @xtuchyna, if you need some more specific filter, you can create a generalized one, that take repos active with a certain manager active (set to True in the table)

Reply:
The `mi` should be able to autodetect which managers are active and which not, however in future it can be reasonable to really inspect if the returned active managers are really active or not.